### DISCUSSION TITLE
Questions regarding plotting in MTEX

### DISCUSSION THREAD

**Discussion by AdamLeaghj** [mtex-toolbox/mtex#1716] [Ask Anything] [answered]
Hi everyone, 

I have 2 minor questions but couldn't find solutions from anywhere so just wanted to lump them into one thread here. Any solution for each of them is very much welcome. Thanks in advance for your help.

1. There is an error popped up when I was trying to plot a band contrast figure. It returned the below error. I think there is no bc class in my ebsd file (using a .ang file).
```
plot(ebsd,ebsd.bc)
colormap gray
mtexColorbar
```
![image](https://user-images.githubusercontent.com/118141541/212188238-20ebea2a-00ee-4692-aa3d-8aae7872705f.png)

2. How to change the microbar font size? I have to manually adjust the font size in one of the settings of figure, so how to set it in my code?
![image](https://user-images.githubusercontent.com/118141541/212187633-d613310e-20ec-453f-b108-160b4de17dcf.png)

**Comment by zmichels** [comment, marked as answer]
1. You are correct that there is no band contrast value in an .ang file or EDAX system... so there will not be a 'bc' property with your imported EBSD data. However, the similar metric from the EDAX system is called "image quality" and the property after .ang import is 'iq'... and you access it with ebsd.iq

**Reply by AdamLeaghj** [reply]
Thanks for your help. It worked perfectly. Do you have any idea for question 2?

**Reply by zmichels** [reply]
I don't, unfortunately. One of the developers may chime in hopefully with some details for you.

**Comment by zmichels** [comment]
band contrast is recorded by Oxford Instruments systems (maybe others, too?) and those file formats are typically .CTF (or a pair of .CPR and .CRC files).

**Reply by AdamLeaghj** [reply]
I deleted the last comment since I figured it out. Just see your comment here. Thanks for your help.

**Comment by lfgmorales** [comment]
Hi

Band contrast only exist on Oxford Instrument files, either *.ctf, *.cpr/crc, etc. You will bot have band contrast in your file because like Zach said, it is from edax, but they are pretty much equivalent (like zach said as well)

The image quality you can read here

<https://edaxblog.com/2016/03/15/intelligent-use-of-iq/>
[image4.png]
Intelligent Use of IQ<https://edaxblog.com/2016/03/15/intelligent-use-of-iq/>
edaxblog.com<https://edaxblog.com/2016/03/15/intelligent-use-of-iq/>




But you have much more information on google

The white areas you have are not indexed, and it seems it is a matter how ypur non indexed pixels are setup

Cheers

Luiz

Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 12 Jan 2023, at 23:25, AdamLeaghj ***@***.***> wrote:

﻿

Hi, just want to make it clear a little bit. Which file extension is compatible with the bc class? And in terms of "image quality", what is the meaning of high or low values? I got some white areas after plotting with iq class which are supposed to be low-index regions. However, the iq values in these regions seem to be higher.

[image]<https://user-images.githubusercontent.com/118141541/212193694-eb264b6e-fac4-4284-a9ff-bf46d6434410.png>

—
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/1716#discussioncomment-4672407>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5IN6B3CEO6DZBWWOIYDDWSCAFRANCNFSM6AAAAAATZXH5RM>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/1716#discussioncomment-4672407", "url": "https://github.com/mtex-toolbox/mtex/discussions/1716#discussioncomment-4672407", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

**Reply by AdamLeaghj** [reply]
Thanks Luiz and Zach, I found the same explanation so deleted the last comment but you guys have captured it. Sorry for that.
Do you have any ideas for my 2nd question?

**Reply by kilir** [reply]
Hi,
have a look here;
https://github.com/mtex-toolbox/mtex/discussions/1298#discussioncomment-1763654

Cheers,
Rüdiger

**Reply by AdamLeaghj** [reply]
Thanks Rudiger!
